### DISCUSSION TITLE
fullscreen to use it for video streaming

### DISCUSSION THREAD

**Discussion by puppinoo** [Meli10/Rocket-Browser#3] [Q&A] [answered]
Hi,
Subject basically says all. I'm looking for a very, very lightweight browser to use with streaming services like paramountplus.
This one seems to work but I cant put it in fullscreen mode.
Is ther a way to do that?

Thanks.

**Comment by Meli10** [comment, marked as answer]
Hi there!

Thank you for reaching out. I’m excited to hear you are fond of Rocket Browser. I see you’re having trouble with expanding the video window to watch streaming videos online in full screen. Fear not, I’m working on a fox as we speak!

While I’m working on this, if you’re inclined to do so, I have an “issue template” as one of the files on my GitHub. When you open it, you will a feature suggestions template and issues template. If you select the feature suggestion and create a ticket, not only can I keep you updated through this ticket, but it also shows activity on my end with the rocket-browser repo! If not, no worries, the feature will be added and uploaded as soon as possible.

Thanks for the support,
Anthony

On Dec 4, 2023, at 10:11 AM, puppinoo ***@***.***> wrote:

﻿

Hi,
Subject basically says all. I'm looking for a very, very lightweight browser to use with streaming services like paramountplus.
This one seems to work but I cant put it in fullscreen mode.
Is ther a way to do that?

Thanks.
